packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -23,6 +23,27 @@ export interface ClusterProps {
    * The VPC where your ECS instances will be running or your ENIs will be deployed
    */
   readonly vpc: ec2.IVpc;
+
+  /**
+   * The service discovery namespace created in this cluster
+   *
+   * @default none
+   */
+  readonly defaultCloudMapNamespace?: NamespaceOptions;
+
+  /**
+   * The name of the autoscaling group to be added
+   *
+   * @default none
+   */
+  readonly autoscalingGroupName?: string;
+
+  /**
+   * The ec2 capacity to add to the cluster
+   *
+   * @default none
```

Comment:
Use `-` as a value and describe the default behavior and reference the method:

```ts
@default - no EC2 capacity will be added, you can use `addCapacity` to add capacity later.
```

Reply:
Addressed.